blog/blog.html

```diff
@@ -1,52 +1,22 @@
 <!DOCTYPE html>
-
-<html class="no-js" lang="en">
+<html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta name="description" content="" />
-    <meta name="author" content="" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
I think this is not really necessary anymore. See: [this thread](https://hinditutor.in/qa/21018/what-does-meta-http-equiv-x-ua-compatible-content-ie-edge-do)